config/crd/bases/config.gatekeeper.sh_configs.yaml

```diff
@@ -38,6 +38,8 @@ spec:
                   properties:
                     excludedNamespaces:
                       items:
+                        description: PrefixWildcard is a string that supports globbing at its end.
```

Comment:
This description should be targeted at consumers of the API, so no "PrefixWildCard", maybe some guidance as to correct inputs and what they do.

Reply:
So far I haven't been able to find a way to do this with `kubebuilder`.  It defaults to using the description that's written on the custom `PrefixWildcard` type in `pkg/util/prefix_wildcard.go`.

Removing `PrefixWildcard is a` from this string causes a lint error:

```
pkg/util/prefix_wildcard.go:7:1: exported: comment on exported type PrefixWildcard should be of the form "PrefixWildcard ..." (with optional leading article) (revive)
// a string that supports globbing at its end.
^
```

Rather than add to the jumbled mess of kustomize, I'm going to add a `//nolint:revive` to the comment and call it a day.

LMK what you think of the more in-depth description I added.